src/flatdirectory.js

```diff
@@ -116,7 +116,7 @@ export class FlatDirectory {
         return false;
     }
 
-    async download(key) {
+    async fetchData(key) {
```

Comment:
why not name this function to downloadSync() ?

Reply:
Because this function is actually 'Async' and 'download' is also 'Async', if you want to execute other content after getting the result, you need to use await in the calling place.
`async downloadAsync` is a bit repetitive, so a different name was chosen to indicate fetching data.